tests/rpmmacro.at

```diff
@@ -990,3 +990,25 @@ xxx
 error: bad
 ])
 AT_CLEANUP
+
+AT_SETUP([bcond macro])
+AT_KEYWORDS([macros])
+AT_CHECK([
```

Comment:
Move this test into the other one in rpmbuild.at - the macro tests are more about testing the macro language and its builtins. There can be multiple AT_CHECK sections inside AT_SETUP-AT_CLEANUP pair, this will fit nicely in the build-side test.